Packs/Hoxhunt/ReleaseNotes/2_0_0.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I don't know why I didn't noticed this before, but you can remove this file because it's not breaking anything, customer that already using the v1 integration will can continue to use it until they're updating to v2.